src/config.js

```diff
@@ -16,6 +16,15 @@ export const AXE_TEST_TIMEOUT = process.env.REACT_AXE_TEST_TIMEOUT || 20000;
 
 export const GEOJSON_MAX_SIZE = process.env.REACT_GEOJSON_MAX_SIZE || 1000000;
 
+export const SHARED_DATA_MAX_SIZE =
+  process.env.REACT_SHARED_DATA_MAX_SIZE || 10000000;
```

Comment:
suggestion: for these large values, add a comment with the human-readable size, i.e `// 10 MB`